datameta/templates/layout.pt

```diff
@@ -63,8 +63,11 @@
         }
 
         function init_widescreen() {
-
-            widescreenCookie = document.cookie.split("; ").find(row=>row.startsWith("widescreen=")).split('=')[1];
+            if(document.cookie) {
+                widescreenCookie = document.cookie.split("; ").find(row=>row.startsWith("widescreen=")).split('=')[1];
+            } else {
+                widescreenCookie = "false";
+            }
```

Comment:
It would be great to fix the undefined error at some point 😆 , but this doesn't fix it I believe. `document.cookie` is always true since the server sets the `datameta` cookie.

Reply:
It's the second `split()` call that triggers `Cannot read property 'split' of undefined`